ngrok-java/src/main/java/com/ngrok/net/ConnectionSocketImpl.java

```diff
@@ -22,11 +22,11 @@ protected void setConnection(Connection connection) {
 
     @Override
     protected InputStream getInputStream() throws IOException {
-        return new ConnectionInputStream(connection, 1024);
+        return new ConnectionInputStream(connection, 4096);
```

Comment:
Should we define this as a const?

```
    private static final int STREAM_BUFFER_SIZE = 4096;
```

Reply:
🤔 maybe? They are not intended/necessarily of the same size

Reply:
actually, I'd rather have it like this, instead of another indirection 🤷